src/index.ts

```diff
@@ -109,6 +159,21 @@ async function init() {
 	}
 }
 
+function isGithubCliInstalled() {
+	try {
+		execSync("gh");
+		return true;
+	} catch (error) {
+		return false;
+	}
+}
+
+function initRepoAndCommit() {
+	execSync("git init", { stdio: "inherit" });
+	execSync("git add .", { stdio: "inherit" });
+	execSync('git commit -m "Initial commit"', { stdio: "inherit" });
```

Comment:
I think maybe it looks better if we hide the output of these commands. After all, we're not committing anything the user has typed, so I thinks it's ok if it's a little less explicit.

```suggestion
	execSync("git init");
	execSync("git add .");
	execSync('git commit -m "Initial commit"');
```